domains/_discord.jirafey.json

```diff
@@ -1,7 +1,6 @@
 {
     "owner": {
-        "username": "jirafey",
-        "email": "jirafey@tuta.io"
+        "username": "jirafey"
```

Comment:
You should add atleast another social if you want to remove the mail

Reply:
@Stef-00012